### ISSUE TITLE
74HC14 vs 74LS14 vs 74HCT14

### ISSUE THREAD

**Issue by mmatt80** [redhawk668/Atari-CO60472-Delay-Line-Replacement#2]
In the BOM, you call for a 74LS14 or 74HC14.  I built one and used a 74HC14 and was having issues with it.  I was having a hard time finding a 30pF cap, so used a 33pF instead at c3.  I'm not sure if I worked it out correctly, but I swapped R5 for a 560R resistor to hopefully get the timing back to were it should be.  This still didn't work, and knowing that HC isn't TTL compatible, I found a spare 74LS14 on a spare expansion board I had.  Put this in and the atari is now working as expected!  So I think you might have meant 74HCT14 instead of 74HC14?  

Thanks for you work on this project, it helped to save another atari!

**Comment by redhawk668** [comment]
Indeed, my bad. It should be a 74HCT14 or 74LS14. 74HC14 is not TTL compatible, so the timing will not be correct.
